templates/prometheus.go

```diff
@@ -34,42 +35,48 @@ var resourceSelector = metav1.LabelSelector{
 	},
 }
 
+const (
+	//TODO: Construct monitoring dns name dynamically based on AWS and IBM Cloud Monitoring regions
+	MonitoringDNSName = "ingest.us-south.monitoring.cloud.ibm.com"
+)
+
 var (
 	KubeRBACProxyPortNumber             int    = 9339
 	PrometheusServingCertSecretName     string = "prometheus-serving-cert-secret"
 	PrometheusKubeRBACPoxyConfigMapName string = "prometheus-kube-rbac-proxy-config"
+	MonitoringURL                       string = fmt.Sprintf("https://%s/prometheus/remote/write", MonitoringDNSName)
 )
 
-// var metrics = []string{
-// 	"job:ceph_versions_running:count",
-// 	"job:ceph_pools_iops_bytes:total",
-// 	"job:ceph_pools_iops:total",
-// 	"job:kube_pv:count",
-// 	"job:ceph_osd_metadata:count",
-// 	"ceph_health_status",
-// 	"ceph_cluster_total_used_raw_bytes",
-// 	"ceph_cluster_total_bytes",
-// 	"cluster:kubelet_volume_stats_used_bytes:provisioner:sum",
-// 	"cluster:kube_persistentvolumeclaim_resource_requests_storage_bytes:provisioner:sum",
-// }
+var metrics = []string{
+	"job:ceph_versions_running:count",
+	"job:ceph_pools_iops_bytes:total",
+	"job:ceph_pools_iops:total",
+	"job:kube_pv:count",
+	"job:ceph_osd_metadata:count",
+	"ceph_health_status",
+	"ceph_cluster_total_used_raw_bytes",
+	"ceph_cluster_total_bytes",
+	"cluster:kubelet_volume_stats_used_bytes:provisioner:sum",
+	"cluster:kube_persistentvolumeclaim_resource_requests_storage_bytes:provisioner:sum",
+}
```

Comment:
Why these metrics? where did the requirement for these come from (in the context of the new agent)?
Also, I think the most important metric is missing.

Reply:
These are the metrics that were exposed to RHOBS in the deployer

Reply:
@nb-ohad I can think of two metrics right now one is `agent_installation_status` and `offering_installation_status`, what other metrics are expected to be exposed?

Reply:
Why do we need these right now? we are missing a lot of infrastructure to support these. I will keep the entire list in a comment for this PR

Reply:
Okay, So what metrics do we need to expose from the agent side for now?

Reply:
none. all work on specific metrics to come later on based on actual use cases